web/README.md

```diff
@@ -40,3 +40,9 @@ npm install
 make dev
 make server
 ```
+
+The workflow for interactive development of just one app in debug mode:
```

Comment:
I couldn't figure out how to use the makefile to just rebuild an app when the code has changed, so I resorted to calling `build-wasm` directly. Then I expected `make dev` to copy the files over, but they didn't seem to, so this is where I landed to test things out.

Reply:
`Then I expected make dev to copy the files over, but they didn't seem to, so this is where I landed to test things out.`

What do you mean by "copy the files over" which files? and copied to where?

Reply:
`web/src/abstreet/wasm_pkg` and `build/dist/abstreet/pkg` seem to be copies. I was looking at

```
build/dist/%.html: src/web_root/*.html
        # just copy everything... we could do something more nuanced
        cp -r src/web_root/* build/dist
```

but I think this is just copying over HTML files. I need to go properly follow a tutorial and learn makefiles finally. :(

Reply:
Oh, you know what, I built this while revving on the JS code, not while revving on the rust code, and didn't account for that use case.

I'll take a look at making it work in a way that's facilitates that workflow.

Reply:
(the basic issue is that we're not looking for any changes in the rust code - just the presence of the wasm directory. Since the wasm directory exists, we assume you're good to go).